liquibase-core/src/main/java/liquibase/database/AbstractJdbcDatabase.java

```diff
@@ -234,6 +234,11 @@ public int getDatabaseMinorVersion() throws DatabaseException {
         }
     }
 
+    @Override
+    public String getDatabaseFullVersion() throws DatabaseException {
+        return null;
+    }
```

Comment:
if no other database type cares about this information, then probably it should not be on Database/AbstractJdbcDatabase level, but moved to Postgres.

second, looks like 'fullVersion' is not needed for Postgres (its just temp step to evaluate **isEdb** check)

third, `boolean` interface for **idEdb** type check is not extendable.
if, in future, you need to discriminate between Aurora and Community...
please choose between string, enum or complex class that encapsulates DbInfo data (like versions, types, compatibility mods, etc)

Reply:
1. I wanted to keep robustness for future. This is a library and soon we can use this method for Aurora Oracle
2. We are going to get rid of Postgres DbDef type -> https://datical.atlassian.net/browse/DAT-2338
3. Agree, I'll refactor it to ENUM